core/modules/path/path.admin.inc

```diff
@@ -123,14 +123,15 @@ function path_admin_edit($path = array()) {
  * @see path_admin_form_delete_submit()
  */
 function path_admin_form($form, &$form_state, $path = array('source' => '', 'alias' => '', 'langcode' => LANGUAGE_NONE, 'pid' => NULL)) {
+  $site_config = config('system.core');
```

Comment:
Both occurrences of config_get() actually use the same config item, so it would be possible to:
```
$clean_urls = config_get('system.core', 'clean_url');
```
And use this variable in both places. There's no other config item used in this function.

Reply:
Done!